protocol/src/test/kotlin/tbdex/sdk/protocol/TbdexTestVectorsProtocol.kt

```diff
@@ -23,36 +24,36 @@ class TbdexTestVectorsProtocol {
   fun parse_close() {
     val vector = TestVectors.getVector("parse-close.json")
     assertNotNull(vector)
-    testNonErrorMessageTestVector<Close>(vector)
+    testSuccessMessageTestVector<Close>(vector)
   }
 
   @Test
   fun parse_order() {
     val vector = TestVectors.getVector("parse-order.json")
     assertNotNull(vector)
-    testNonErrorMessageTestVector<Order>(vector)
+    testSuccessMessageTestVector<Order>(vector)
   }
 
   @Test
   fun parse_orderstatus() {
     val vector = TestVectors.getVector("parse-orderstatus.json")
     assertNotNull(vector)
-    testNonErrorMessageTestVector<OrderStatus>(vector)
+    testSuccessMessageTestVector<OrderStatus>(vector)
   }
 
 
   @Test
   fun parse_quote() {
     val vector = TestVectors.getVector("parse-quote.json")
     assertNotNull(vector)
-    testNonErrorMessageTestVector<Quote>(vector)
+    testSuccessMessageTestVector<Quote>(vector)
   }
 
   @Test
   fun parse_rfq() {
```

Comment:
For the benefit of other reviews: This is one of the two tests that we expect to fail for this PR. The only other test that should fail is `ValidatorTest > validateFailsWithInvalidRfqData()`. All other tests should (and do) pass.

 Context: #221